src/Syntax.jl

```diff
@@ -968,16 +970,38 @@ end
 
 
 """
-feet(block :: Expr)
+(block :: Expr)
 
-Takes as argument an expression of the form A => B and creates a foot (StockAndFlow0).
+Takes as argument an expression of the form A => B, C => (), D => E and creates a single foot from it.
+Can also take arguments of form A => B.
 """
 function create_foot(block::Expr)
-    @match block begin
-        :(()              => ())              => foot((), (), ())
-        :($(s :: Symbol)  => ())              => foot(s, (), ())
-        :(()              => $(sv :: Symbol)) => foot((), sv, ())
-        :($(s :: Symbol)  => $(sv :: Symbol)) => foot(s, sv, s => sv)
+    @match block.head begin
+        :tuple => begin #TODO: Make more efficient
```

Comment:
@Xiaoyan-Li might know more about the order of stocks, svs and links mattering in feet.  I know position currently matters for equality (eg, foot((:A,:B), (), ()) != foot((:B, :A), (), ())), but I'm not sure if it matters for anything else.

Reply:
We could use an [accumulator](https://juliacollections.github.io/DataStructures.jl/latest/accumulators/) (or just Dict{Pair{Symbol, Symbol}, Int64}) for foot_ssv?  Don't know if that'd be any faster.

Reply:
I'm hesitant to update when I don't know if order matters

Reply:
I switched to using Vectors + calling `unique()` on them instead and it's also pretty quick to use mutable vectors and a for loop but it also seems to preserve the order. I think it may even be faster than the set impl.

On length 148:
```
julia> (a, b) = runbench2(e)
mut vec + unique
  0.000253 seconds (961 allocations: 103.328 KiB)
maps + filters
 25.381629 seconds (17.29 M allocations: 1.019 GiB, 1.42% gc time)
julia> a == b
true
```

Compared with set impl on length 1470:
```
mut vec + unique
  0.001416 seconds (9.40 k allocations: 886.844 KiB)
mut vec + set
  0.001290 seconds (12.41 k allocations: 890.625 KiB)
```

Reply:
Updated to use aforementioned pseudocode but with vectors.